docs/lake/design.md

```diff
@@ -542,11 +546,11 @@ in the index group `IndexGroupEx`.
 #### Aggregation Rule
 
 An aggregation rule allows the creation of any index keyed by one or more fields
-(primary, second, etc) typically the result of an aggregation.
+(primary, second, etc.), typically the result of an aggregation.
 The aggregation is specified as a Zed query.
 For example,
 ```
-zed lake index create IndexGroupEx agg "count() by _path"
+zed lake index create IndexGroupEx agg "count() by field"
```

Comment:
This doc had already been thoroughly de-Zeeked, so I figure we might as well stick with that.